src/bundle/Resources/public/scss/_alloyeditor-link-edit.scss

```diff
@@ -1,92 +1,127 @@
+    column-gap: calculateRem(16px);
+    grid-template-areas:
+        'udw actions'
+        'info actions';
 
     &__row {
         display: flex;
         align-items: flex-end;
 
-        &:not(:last-child) {
-            margin-bottom: 1rem;
+        &:first-child {
+            margin-bottom: calculateRem(8px);
         }
 
-        &.ez-ae-edit-link__row--actions {
-            justify-content: flex-end;
+        &--udw {
+            grid-area: udw;
+        }
+
+        &--info {
+            grid-area: info;
+        }
+
+        &--actions {
+            grid-area: actions;
+            padding: 0 0 0 calculateRem(14px);
+            border-left: calculateRem(2px) solid $ez-ground-base-dark;
         }
     }
 
-    .ez-ae-edit-link__block {
-        &.ez-ae-edit-link__block--url {
+    &__block {
+        &--udw {
+            display: flex;
+            flex-direction: column;
+        }
+
+        &--url {
             position: relative;
             display: flex;
             flex-direction: column;
             width: 25rem;
         }
 
-        &.ez-ae-edit-link__block--title,
-        &.ez-ae-edit-link__block--target {
+        &--title,
+        &--target {
             display: flex;
             flex-direction: column;
 
             .ez-ae-edit-link__text {
                 margin-bottom: 0.3rem;
+                font-weight: bold;
             }
         }
 
-        &.ez-ae-edit-link__block--title {
-            width: 24rem;
-            margin-right: 1rem;
+        &--title {
+            width: calculateRem(464px);
+            margin-right: calculateRem(16px);
         }
 
-        &.ez-ae-edit-link__block--target {
+        &--target {
             .ez-ae-edit-link__choice {
-                height: 2.25rem;
+                border: calculateRem(2px) solid $ez-color-secondary;
+                padding: calculateRem(2px);
+                border-radius: calculateRem(4px);
 
                 .ez-ae-edit-link__label {
                     margin-bottom: 0;
 
-                    span {
-                        background-color: $ez-color-base-dark;
-                        height: 2.25rem;
-                        padding: .7rem .5rem;
-                        display: block;
-                        color: $ez-black;
+                    &:not(:last-child) {
+                        .ez-btn-ae__icon-wrapper {
+                            margin-right: calculateRem(8px);
+                        }
                     }
 
-                    input:checked + span {
-                        background-color: $ez-color-base-light;
-                        color: $ez-white;
+                    .ez-icon {
+                        fill: $ez-color-secondary;
+                        width: calculateRem(20px);
+                        height: calculateRem(20px);
                     }
 
-                    [type="radio"] {
-                        display: none;
+                    .ez-btn-ae__icon-wrapper {
```

Comment:
Icons within Open Tab Switcher look like they have extra space below them, maybe adding `flex` properties for centering them would work here.